nservicebus/containers/index.md

```diff
@@ -16,6 +20,38 @@ NServiceBus automatically registers all its components as well as user-implement
 
 NServiceBus has a built-in container (currently an ILMerged version of Autofac) but it can be replaced by any other container.
 
+Using an external container is usefull when:
+
+* A external container is already present and you want types that are registered in that container to be injected in your messages handlers or other types that NServiceBus creates internally.
```

Comment:
> Using an external container is usefull when an external container is already present

Isnt this effectively inception 
![tumblr_lpf62pedwi1qzdni0o1_400](https://cloud.githubusercontent.com/assets/122666/8224674/ee4b8b98-15cd-11e5-86a9-92cc3a56c772.gif)